src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.GuardedCallsTests.cs

```diff
@@ -3515,6 +3515,55 @@ Class Test
         M2()
     End Sub
 
+    <SupportedOSPlatform(""Windows10.1.2.3"")>
```

Comment:
That might be too much work, but it'd be nice to add tests for the various rules impacted by having this change.

Reply:
> That might be too much work

That is what i thought 😛, ok, i will add some more